src/test/java/com/cdancy/bitbucket/rest/features/DefaultReviewersApiLiveTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.features;
+
+import com.cdancy.bitbucket.rest.BaseBitbucketApiLiveTest;
+import com.cdancy.bitbucket.rest.GeneratedTestContents;
+import com.cdancy.bitbucket.rest.domain.branch.Matcher;
+import com.cdancy.bitbucket.rest.domain.defaultreviewers.Condition;
+import com.cdancy.bitbucket.rest.domain.pullrequest.User;
+import com.cdancy.bitbucket.rest.options.CreateCondition;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Test(groups = "live", testName = "DefaultReviewersApiLiveTest", singleThreaded = true)
+public class DefaultReviewersApiLiveTest extends BaseBitbucketApiLiveTest {
+
+
+    private GeneratedTestContents generatedTestContents;
+
+
+    @BeforeClass
+    public void init() {
+        generatedTestContents = initGeneratedTestContents();
+    }
+
+    @Test
+    public void testListConditionsOnEmptyRepo() {
+        List<Condition> conditionList = api().listConditions(generatedTestContents.project.key(), generatedTestContents.repository.slug());
+        assertThat(conditionList).isEmpty();
+    }
+
+
+    @Test(dependsOnMethods = {"testListConditionsOnEmptyRepo"})
+    public void testCreateCondition() {
+        Long requiredApprover = 1L;
+        Matcher matcherSrc = Matcher.create(Matcher.MatcherId.ANY, true);
+        Matcher matcherDst = Matcher.create(Matcher.MatcherId.ANY, true);
+        List<User> listUser = new ArrayList<>();
+        listUser.add(User.create("test", "test@test.com", 1, "test", true, "test", "NORMAL"));
```

Comment:
Will this work if you just add any user? If not we can always use the default logged in user we are running these tests with.